### ISSUE TITLE
What is the status of this project ?

### ISSUE THREAD

**Issue by germgerm** [rufanov/CsQueryCore#1]
Really missing CsQuery in core.

**Comment by rufanov** [comment]
There is only one thing left - fixing code that handles CSS, because in original project it's heavily depends on bugs and flaws in  JSON parsing via System.Web.Script.Serialization.JavaScriptSerializer, that don't exists in System.Text.Json or in Newtonsoft.Json - so tests fail while using this libraries.

Currently trying to write some code that will convert "non-really-a-json" to the "standart json that can be parsed using normal json parser"

**Comment by germgerm** [comment]
Is the issue documented somewhere ?
Does cause problems with parsing or serialisation ?

**Comment by rufanov** [comment]
@germgerm , no, not documented, and it's problem with parsing - parsing elements with inlined style-properties are broken for now... also there is little problem with encodings, that have quick fix..

But i can only continue on weekend, because of work ( I have plans to release package to nuget as soon as this problems will be fixed

**Comment by germgerm** [comment]
Thanks for the response and the effort.
Is there a particular area where you could use a hand ?
